app/term/term.html

```diff
@@ -147,7 +147,7 @@ <h3 class="term-section-header"><span uib-tooltip-template="'childterms.html'" t
 
             <th>Relationship to </br>{{termModel.termId}}</th>
           </tr>
-          <tr ng-repeat="childRelation in termModel.childTermsRelations">
+          <tr ng-repeat="childRelation in termModel.childTermsRelations | limitTo:childTermsPageSize">
```

Comment:
Did you miss on purpose the Synonyms sections below the Overview and above the chart?